website/docs/components/carousel/carousel-a11y.md

```diff
@@ -13,7 +13,7 @@ Table: Keyboard support
 | Key                         | Function                                                                                                                                                                           |
 | --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `Tab`, `Shift + Tab`        | Moves focus through interactive elements in the carousel.                                                                                                                          |
-| `Enter`, `Space`            | Opens slide in modal window if `zoom` property is set to `true`.                                                                                                                   |
+| `Enter`                     | Opens slide in modal window if `zoom` property is set to `true`.                                                                                                                   |
```

Comment:
hmmm it might be an issue. It has role=button and buttons are supposed to activate on Space

Reply:
Yep, it was some strange behaviour, I've wrote Michael about. If it will finally works without Space scrolling the page, I'll leave the `Space`.